src/modules/app/createReduxStore.js

```diff
@@ -8,11 +8,14 @@ import endpointReducers from '../endpoint/reducers'
 import { createResponsiveStateReducer, responsiveStoreEnhancer } from 'redux-responsive'
 import defaultRoutesMap from './routesMap'
 import onBeforeChange from './onBeforeChange'
+import queryString from 'query-string'
 
 const createReduxStore = (createHistory, initialState = {}, routesMap = defaultRoutesMap) => {
   const history = createHistory()
 
-  const {reducer, middleware, enhancer} = connectRoutes(history, routesMap, {onBeforeChange: onBeforeChange})
+  const {reducer, middleware, enhancer} = connectRoutes(history, routesMap,
+    {onBeforeChange: onBeforeChange, querySerializer: queryString}
```

Comment:
So query-string is like a dependency for redux-first-router?

Reply:
nope, this is just so that you can write <Link to={'path?withQuery'}/>  instead of <Link to={type={} payload={} query={}/>

Reply:
Do we need this? I like the second way more because you do not need to build strings.

Reply:
Okay, changed it in #181. query-string is needed anyway.